### ISSUE TITLE
Local child class, can't access parent functions

### ISSUE THREAD

**Issue by LoneDespair** [godotengine/godot#46287]
**Godot version:**
3.2.3-stable

**OS/device including version:**
Windows 10

**Issue description:**
extending a class within the same file, causes the child class to not detect its parents functions, (Sometimes it does, but will eventually break again)

**Steps to reproduce:**
```gdscript
class_name ParentClass

func hi() -> void:
	printt("Hello world")

class ChildClass extends ParentClass:
	func _init() -> void:
		hi()
		# hi isn't declared in the current class
```

**Minimal reproduction project:**

**Comment by hilfazer** [comment]
You can't call non-static function like this. You need an object of the outer class. Inner class in GDScript does not mean same thing as in Java.

**Comment by KoBeWi** [comment]
This is fixed in 4.0
